docs/crictl.md

```diff
@@ -0,0 +1,160 @@
+CRICTL Client User Guide
+========================
+This document presumes you already have `containerd` and `cri-containerd`
+installed and running.
+
+This document is for developers who wish to debug/inspect and manage their pods,
+containers, and container images.
+
+Before generating issues against this document, `containerd`, `cri-containerd`,
+or `crictl` please make sure the issue has not already been submitted.
+
+## Install crictl
+If you have not already installed crictl please install the version compatible
+with the `cri-containerd` you are using. The current version is specified
+[here](./hack/version). A helper command has been included to install the
+dependencies at the right version for you:
+```shell
+# make install.deps
```

Comment:
may be a note ? That install.deps does do a whole bunch of things other than crictl installation ?

Reply:
thought about it.. they'll figure that out :-)

Reply:
and we might want to split em up..

Reply:
Actually we package `crictl` into our release tarball, can we also mention that?
If you are a user, your deployement should have install `crictl` for you. If not, get it from release tarball.
If you are a developer, use `make install.deps`.

We should also mention the configuration file `/etc/crictl.yaml`. We also have it in the tarball.

Reply:
sgtm